src/client/app/utils/hasPermissions.ts

```diff
@@ -21,5 +21,7 @@ export function hasPermissions(user: UserRole, compareTo: UserRole): boolean {
  * @returns Whether or not user is an admin
  */
 export function isRoleAdmin(user: UserRole): boolean {
+	// TODO Already Converted to a selector
```

Comment:
The only place this is used is src/client/app/components/HeaderButtonsComponent.tsx. The comment seems out of date since that function does not exist. I think we should purge the last use and get rid of this put want to check with you on the best way to do that.

Reply:
Yeah, these are essentially selectors. This file is legacy, and will be deleted.